test/functional/test_framework/test_framework.py

```diff
@@ -378,7 +378,8 @@ def setup_nodes(self):
             for w in wallets[i]:
                 wallet_name = w.split('=', 1)[1]
                 n.createwallet(wallet_name=wallet_name, descriptors=self.options.descriptors)
-        self.import_deterministic_coinbase_privkeys()
+        if self.chain != "signet":
```

Comment:
Because the privkeys cannot be imported due to signet having a different prefix. (`Invalid private key encoding (-5)`)

Reply:
Probably another reason why it would be better to have the same prefix. See also:

* https://github.com/bitcoin/bitcoin/pull/18267#discussion_r469713812
* #12314

Reply:
Switched to using testnet prefixes (including bech32 one).

Reply:
After switching I noticed that `rust-bitcoin` relies on bech32 prefixes to be unique per network. This change breaks that assumption.

Reply:
It also breaks the assumption that each prefix (pubkey etc) maps to a specific network. Maybe I need to revert this change..